changes/2693.bugfix.rst

```diff
@@ -0,0 +1,14 @@
+Implement open() for LoggingStore
+LoggingStore is now a generic class.
+Add _is_open property and setter for WrapperStore
```

Comment:
Since this is private API, no need to put it in the changelog (which is user facing).
```suggestion
```